pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -97,8 +98,8 @@ func (m *Manager) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAdmitR
 	if nodeShuttingDown {
 		return lifecycle.PodAdmitResult{
 			Admit:   false,
-			Reason:  nodeShutdownReason,
-			Message: nodeShutdownNotAdmitMessage,
+			Reason:  nodeShutdownNotAdmittedReason,
```

Comment:
should the reason could be simply `NodeShutdown`? as the reason for pod termination is node shutdown.

Reply:
Sure, that's a good change;

I also changed the message to reflect the new reason:
	nodeShutdownReason             = "Terminated"
	nodeShutdownMessage            = "Pod was terminated in response to node shutdown."
	nodeShutdownNotAdmittedReason  = "NodeShutdown"
	nodeShutdownNotAdmittedMessage = "Pod was rejected in response to node shutdown."